README.md

```diff
@@ -61,8 +61,12 @@ environments:
     restart_command: etc/init.d/daemon restart
 
 slack: # optional
-  webhook: https://hooks.slack.com/services/xxx/xxx # no more custom settings for now, please select emoji, name etc. via Slack
+  token: slack token
+  channel: bot_checks
   appname: AppName
+  start_emoji: rocket
+  finish_emoji: heart
```

Comment:
it'd be cool to have some default emojis for users who don't want to define their own ones, also please remember about the format `:<emoji name>:` - seems like it won't handle it properly with such config ;(